fdbserver/Knobs.cpp

```diff
@@ -177,7 +177,7 @@ ServerKnobs::ServerKnobs(bool randomize, ClientKnobs* clientKnobs) {
 	init( MAX_TEAMS_PER_SERVER,           5*DESIRED_TEAMS_PER_SERVER );
 	init( DD_SHARD_SIZE_GRANULARITY,                         5000000 );
 	init( DD_SHARD_SIZE_GRANULARITY_SIM,                      500000 ); if( randomize && BUGGIFY ) DD_SHARD_SIZE_GRANULARITY_SIM = 0;
-	init( DD_MOVE_KEYS_PARALLELISM,                               20 ); if( randomize && BUGGIFY ) DD_MOVE_KEYS_PARALLELISM = 1;
+	init( DD_MOVE_KEYS_PARALLELISM,                               10 ); if( randomize && BUGGIFY ) DD_MOVE_KEYS_PARALLELISM = 1;
```

Comment:
The commit message indicates this is being changed for a very limited case (switching from triple to double replication). Is this going to cost us performance in the general case, and is worth it for this rare event? In other words, is there something more targeted we could do that makes sense?